apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/sink/AbstractClaimRdaSink.java

```diff
@@ -129,6 +144,35 @@ private void mergeBatch(Iterable<RdaChange<TClaim>> changes) {
     }
   }
 
+  private long maxSequenceInBatch(Collection<RdaChange<TClaim>> claims) {
+    OptionalLong value = claims.stream().mapToLong(RdaChange::getSequenceNumber).max();
```

Comment:
would this not always be the last claim on the stream?

Reply:
Yes it should be.  This was a bit of defensive coding.